src/main/java/edu/kis/powp/jobs2d/drivers/factories/FigureFactory.java

```diff
@@ -0,0 +1,76 @@
+package edu.kis.powp.jobs2d.drivers.factories;
+
+import edu.kis.powp.jobs2d.drivers.commands.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.commands.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.commands.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.commands.SetPositionCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FigureFactory {
+
+    public enum Figure {
+        RECTANGLE,
+        TRIANGLE
+    }
+
+    private static class Rectangle {
+        public static final int x1 = 200;
+        public static final int y1 = 200;
+
+        public static final int x2 = 200;
+        public static final int y2 = -200;
+
+        public static final int x3 = -200;
+        public static final int y3 = -200;
+
+        public static final int x4 = -200;
+        public static final int y4 = 200;
```

Comment:
Exposing those attributes makes it possible to break Rectangle contract (e.g. returning a diamond or irregular shape)

Reply:
My bad. Its ok :)